test/utils/simple-gu-stack.ts

```diff
@@ -0,0 +1,4 @@
+import { App } from "@aws-cdk/core";
+import { GuStack } from "../../src/constructs/core";
+
+export const simpleGuStackForTesting = () => new GuStack(new App(), "Test", { app: "testing" });
```

Comment:
This is a great idea! Maybe it should be it's own PR though as it's a pretty big diff?

Reply:
Good idea - https://github.com/guardian/cdk/pull/70